src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -202,11 +201,6 @@ public XamlCodeGeneration(GeneratorExecutionContext context)
 				_enableFuzzyMatching = true;
 			}
 
-			_targetPath = Path.Combine(
```

Comment:
We may still need that part for the uno.sourcegeneration side. We'll remove it when we remove Xamarin.

Reply:
Yes let's keep it for !NETSTANDARD2_0 for now.